bba/bba.go

```diff
@@ -174,6 +179,10 @@ func (bba *BBA) Result() (cleisthenes.Binary, bool) {
 	return bba.dec.Value(), bba.dec.Undefined()
 }
 
+func (bba *BBA) Accept() bool {
```

Comment:
What is your intend of `Accept` function?

Is it to show whether bba has started or not? if it is, how about naming it as `Idle()`?

Reply:
good